packages/blockchain-wallet-v4-frontend/src/components/OrderStatus/index.js

```diff
@@ -128,30 +128,30 @@ export const OrderNote = ({ status }) => {
     case PENDING_EXECUTION:
       return (
         <FormattedMessage
-          id='components.orderstatus.note.pending'
+          id='components.orderstatus.note.swappending'
           defaultMessage='Thanks for placing your trade! Funds will usually reach your wallet within 2 hours and we’ll send you a notification when that happens. Keep track of your trade’s progress in the Order History tab.'
         />
       )
     case PENDING_REFUND:
       return (
         <FormattedMessage
-          id='components.orderstatus.note.refunding'
+          id='components.orderstatus.note.swaprefunding'
           defaultMessage='We are now processing a refund for any funds broadcast minus the network fee. Please check your Order History to check the refund status.'
         />
```

Comment:
Not a big deal, but changing the id of a message will require new strings to be translated. If the default message doesn't change its a bit unnecessary.